python/snewpy/snowglobes.py

```diff
@@ -40,7 +40,7 @@
 
 logger = logging.getLogger(__name__)
 
-def generate_time_series(model_path, model_type, transformation_type, d, output_filename=None, ntbins=30, deltat=None):
+def generate_time_series(model_path, model_type, transformation_type, d, output_filename=None, ntbins=30, deltat=None, snmodel_dict={}):
```

Comment:
Could you add a description of this new parameter do the docstring below, please? E.g., something like
```
snmodel_dict : dict
    Additional arguments for setting up the supernova model. See documentation of relevant ``SupernovaModel`` subclass for available options. (Optional.)
```
(And same in `generate_fluence()` below.)

Reply:
Done!